geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/filter/package.scala

```diff
@@ -307,33 +306,7 @@ package object filter {
       case f: Filter => Seq(f)
     }
 
-  /**
-   * Checks the order of properties and literals in the expression
-   *
-   * @param one
-   * @param two
-   * @return (prop, literal, whether the order was flipped)
-   */
-  def checkOrder(one: Expression, two: Expression): Option[PropertyLiteral] =
-    (one, two) match {
-      case (p: PropertyName, l: Literal) => Some(PropertyLiteral(p.getPropertyName, l, None, flipped = false))
-      case (l: Literal, p: PropertyName) => Some(PropertyLiteral(p.getPropertyName, l, None, flipped = true))
-      case (_: PropertyName, _: PropertyName) | (_: Literal, _: Literal) => None
-      case _ =>
-        val msg = s"Unhandled expressions in strategy: ${one.getClass.getName}, ${two.getClass.getName}"
-        throw new RuntimeException(msg)
-
-    }
+  val checkOrder = org.locationtech.geomesa.filter.checkOrder _
 
-  /**
-   * Checks the order of properties and literals in the expression - if the expression does not contain
-   * a property and a literal, throws an exception.
-   *
-   * @param one
-   * @param two
-   * @return
-   */
-  def checkOrderUnsafe(one: Expression, two: Expression): PropertyLiteral =
-    checkOrder(one, two)
-        .getOrElse(throw new RuntimeException("Expressions did not contain valid property and literal"))
+  val checkOrderUnsafe = org.locationtech.geomesa.filter.checkOrderUnsafe _
```

Comment:
Sure, I'll relocate them.  Or, maybe relocate the whole class to org.locationtech.geomesa.filter (same as it was before zcurve)?  The only problem with that is FilterPackageObjectTest depends on TestData in geomesa-accumulo-datastore/src/test.